app/src/main/java/io/simplon/toomanychoco/db/DbConnector.java

```diff
@@ -5,7 +5,7 @@
 import java.sql.SQLException;
 
 public class DbConnector {
-	private static final String JDBC_URL = "jdbc:postgresql://localhost:5432/TooManyChoco";
+	private static final String JDBC_URL = "jdbc:postgresql://localhost:5432/toomanychoco";
```

Comment:
Do not change the database credentials for other peoples.
```suggestion
	private static final String JDBC_URL = "jdbc:postgresql://localhost:5432/TooManyChoco";
```